core/renderers/common/constants.js

```diff
@@ -28,6 +28,108 @@ const {Svg} = goog.require('Blockly.utils.Svg');
 /* eslint-disable-next-line no-unused-vars */
 const {Theme} = goog.requireType('Blockly.Theme');
 
+/**
+ * An object containing sizing and path information about outside corners.
+ * @typedef {{
+ *   topLeft: string,
+ *   topRight: string,
+ *   bottomRight: string,
+ *   bottomLeft: string,
+ *   rightHeight: number
+ * }}
+ */
+let OutsideCorners;
+exports.OutsideCorners = OutsideCorners;
```

Comment:
I'm okay with these values existing but not being available outside of Blockly *yet*, because it feels like there should be a better way to organize and formalize the types. 

Good catch on the typedef in flyout_base. If that should only be used internally, please swap it to being a separate export and *don't* re-export it anywhere.